kubeflow/aws/OWNERS

```diff
@@ -1,3 +1,3 @@
 approvers:
   - jeffwan
-  - jlewi
\ No newline at end of file
+  - mameshini
```

Comment:
Can you please join kubeflow org member list?
@mameshini

Reply:
I will have a clean up for aws owners. Jeremy suggested me to add some contributors like Igor when we had this folder. Since most of codes will move to kfctl golang and we get some regular contributor. This would be changed later.

Reply:
@kunmingg Sure, I submitted PR to join Kubeflow org member list

Reply:
@kunmingg Could you double check it and approve this PR? CLA issue is because of author cherry-pick a different user's PR. Author already sign CLA and this should be ok.